### ISSUE TITLE
Posible blink issue when duplicating the block

### ISSUE THREAD

**Issue by htmgarcia** [publishpress/PublishPress-Blocks#591]
https://github.com/publishpress/PublishPress-Blocks/issues/537#issuecomment-849332024

**Comment by contactashish13** [comment]
@htmgarcia have created a new PR: https://github.com/publishpress/PublishPress-Blocks-Pro/pull/26

**Comment by htmgarcia** [comment]
@contactashish13 the blink issue is still happening.

**Comment by contactashish13** [comment]
@htmgarcia Yes :) This PR was merely a copy of the PR that I closed: https://github.com/contactashish13/PublishPress-Blocks-Pro/pull/1 as per [this comment](https://github.com/publishpress/PublishPress-Blocks/issues/537#issuecomment-858571914)
